omero/sysadmins/limitations.rst

```diff
@@ -43,15 +43,31 @@ user, you cannot make the group into a private group.
 File format support
 -------------------
 
+Large images
+^^^^^^^^^^^^
+
+When you import an image over a certain size, OMERO will generate a pyramid of lower resolution
+images if it doesn't already exist in the file. The threshold size is configurable using
+:property:`omero.pixeldata.max_plane_height` and
+:property:`omero.pixeldata.max_plane_width` but set to 3192x3192 pixels by
+default. However, this process can be very resource-intensive, depending on the size of the
+image as well as the image format and any data compression used, for example see
+`PixelData threads and pyramid generation issues <https://forum.image.sc/t/pixeldata-threads-and-pyramid-generation-issues/49794>`_.
+
+The OMERO pyramid generation process should be considered as deprecated and instead it is recommended
+that users avoid these issues by converting
+their data to `pyramidal OME-TIFF <https://www.openmicroscopy.org/2018/11/29/ometiffpyramid.html>`_
+files before importing into OMERO. A number of suitable tools are available such as
+`bioformats2raw & raw2ometiff<https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html>`_,
+`bfconvert`, `Kheops<https://github.com/BIOP/ijp-kheops>`_, `tifffile`, `aicsimageio`, `libvips` and `QuPath`.
+
 Large images with floating-point pixel data
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 :model_doc:`Pyramids <omero-pyramid/>` of image tiles are currently not
```

Comment:
While the content of this section remains unchanged i.e. floating-point data is not support in the OMERO pyramid generation service, we could add a concluding sentence explaining that floating-point pyramidal OME-TIFF can be generated using the same tools as described above.